### ISSUE TITLE
How many epochs is appropriate for librispeech?

### ISSUE THREAD

**Issue by njuzrs** [zzw922cn/Automatic_Speech_Recognition#19]
I did experiment on librispeech dataset, I have tried 15 epochs when training, but got 0.46 CER and results seems not good. So could you tell me how many epochs is appropriate for librispeech?

**Comment by zzw922cn** [comment]
@njuzrs Since time length of Librispeech is very large, nearly 1000 hours. Epoch is decided by your batch size, learning rate, or optimizers. It is pretty hard to say a definite number of epochs. You can observe the test character error rate, 0.46 is very large, you should keep training until convergence, or make sure CER smaller than 0.10.
